src/store.js

```diff
@@ -64,61 +29,13 @@ import {
   hashContext,
   equalPath,
   getThought,
-  initialState,
   sync,
   syncRemote,
   userAuthenticated
 } from './util.js'
 
-export const appReducer = (state = initialState(), action) => {
-  // console.info('ACTION', action)
-  return Object.assign({}, state, (({
-    alert,
-    authenticate,
-    clear,
-    codeChange,
-    cursorBeforeSearch,
-    cursorHistory,
-    deleteData,
-    dragInProgress,
-    editing,
-    error,
-    existingThoughtChange,
-    existingThoughtDelete,
-    existingThoughtMove,
-    expandContextThought,
-    loadLocalState,
-    modalComplete,
-    modalRemindMeLater,
-    newThoughtSubmit,
-    render,
-    search,
-    searchLimit,
-    selectionChange,
-    setCursor,
-    settings,
-    showModal,
-    showOverlay,
-    hideOverlay,
-    status,
-    thoughtIndex,
-    toggleBindContext,
-    toggleCodeView,
-    toggleContextView,
-    toggleProseView,
-    toggleQueue,
-    toggleSidebar,
-    tutorial,
-    tutorialChoice,
-    tutorialStep,
-
-  })[action.type] || (() => {
-    if (!action.type.startsWith('@@')) {
-      console.error('Unrecognized action:', action.type, action)
-    }
-    return state
-  }))(state, action))
-}
+// middleware
+const middleware = [thunk]
```

Comment:
Please avoid single-use variables and inline the value instead.

Reply:
Just removed middleware variable and used [thunk] directly.